account/src/controller/accountController.js

```diff
@@ -10,18 +14,56 @@ class AccountController {
     });
   };
 
-  static criarAccount = (req, res) => {
+  static criarAccount = async (req, res) => {
+    const senha = req.body.password;
+    const senhaValida = await AccountController.validarSenha(senha);
```

Comment:
Não poderia ter utilizado this.validarSenha ao invés de referenciar o próprio nome da classe? (em geral é o mais comum quando um método da classe é chamado de dentro da própria classe)